docs/HeadlessJSAndroid.md

```diff
@@ -57,4 +57,76 @@ Now, whenever you [start your service][0], e.g. as a periodic task or in respons
 * By default, your app will crash if you try to run a task while the app is in the foreground. This is to prevent developers from shooting themselves in the foot by doing a lot of work in a task and slowing the UI. There is a way around this.
 * If you start your service from a `BroadcastReceiver`, make sure to call `HeadlessJsTaskService.acquireWakelockNow()` before returning from `onReceive()`.
 
+## Example Usage
+```xml
+<receiver android:name=".NetworkChangeReceiver" >
+  <intent-filter>
+    <action android:name="android.net.conn.CONNECTIVITY_CHANGE" />
+  </intent-filter>
+</receiver>
+```
+
+```java
+public class NetworkChangeReceiver extends BroadcastReceiver {
+
+    @Override
+    public void onReceive(final Context context, final Intent intent) {
+        /**
+          This part will be called everytime network connection is changed
+          e.g. Connected -> Not Connected
+        **/
+        if (!isAppOnForeground((context))) {
+            /**
+              We will start our service and send extra info about 
+              network connections
+            **/
+            boolean hasInternet = checkInternet(context);
```

Comment:
Yes you are right. It's just some old leftover code I forgot to change. (originally the function did a bit more)